searchcore/src/tests/proton/proton_config_fetcher/proton_config_fetcher_test.cpp

```diff
@@ -267,16 +267,22 @@ TEST_FF("require that documentdb config manager builds schema with imported attr
     docType->attributesBuilder.attribute[0].name = "imported";
     docType->attributesBuilder.attribute[0].imported = true;
     docType->attributesBuilder.attribute[1].name = "regular";
-
-    const auto &schema = getDocumentDBConfig(f1, f2)->getSchemaSP();
-    EXPECT_EQUAL(1u, schema->getNumImportedAttributeFields());
-    EXPECT_EQUAL("imported", schema->getImportedAttributeFields()[0].getName());
-    EXPECT_EQUAL(1u, schema->getNumAttributeFields());
-    EXPECT_EQUAL("regular", schema->getAttributeFields()[0].getName());
-
-    const auto &attrCfg = getDocumentDBConfig(f1, f2)->getAttributesConfig();
-    EXPECT_EQUAL(1u, attrCfg.attribute.size());
-    EXPECT_EQUAL("regular", attrCfg.attribute[0].name);
+    docType->summaryBuilder.classes.resize(1);
+    docType->summaryBuilder.classes[0].id = 1;
+    docType->summaryBuilder.classes[0].name = "a";
+
+    for (size_t i(0); i < 3; i++) {
+        const auto &schema = getDocumentDBConfig(f1, f2)->getSchemaSP();
+        EXPECT_EQUAL(1u, schema->getNumImportedAttributeFields());
+        EXPECT_EQUAL("imported", schema->getImportedAttributeFields()[0].getName());
+        EXPECT_EQUAL(1u, schema->getNumAttributeFields());
+        EXPECT_EQUAL("regular", schema->getAttributeFields()[0].getName());
+
+        const auto &attrCfg = getDocumentDBConfig(f1, f2)->getAttributesConfig();
+        EXPECT_EQUAL(1u, attrCfg.attribute.size());
+        EXPECT_EQUAL("regular", attrCfg.attribute[0].name);
+        docType->summaryBuilder.classes[0].id = i+2;
```

Comment:
Consider adding a comment on why this is needed

Reply:
Done